packages/vaadin-date-time-picker/src/vaadin-date-time-picker.d.ts

```diff
@@ -1,7 +1,7 @@
-import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
-
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
```

Comment:
**nit:** Sort the mixins in alphabetical order.

Reply:
Done.